qubes-kde-dom0/qubes-kde-dom0.spec

```diff
@@ -23,8 +23,9 @@ Requires: kde-filesystem
 Requires: kde-settings
 Requires: kde-settings-kdm
 Requires: kdelibs >= %{version}
-Requires: kde-workspace
-Requires: kde-workspace-libs
+Requires: kde-workspace-common
+#Requires: kde-workspace
+#Requires: kde-workspace-libs
```

Comment:
I have attempted to address this comment, but what I have can be considered as work-in-progress at best. (There is a need to adapt the KDE 4 customizations to KDE 5 -- that is, aside from the breeze theme.)